Library/Formula/tesseract.rb

```diff
@@ -17,6 +17,7 @@ class Tesseract < Formula
     depends_on "automake" => :build
     depends_on "libtool" => :build
     depends_on "pkg-config" => :build
+    depends_on "cairo" => :build
```

Comment:
Removed and squashed/pushed - seems like the warning only occurs during a `./configure` issue with default clang + OpenMP on current Tesseract HEAD (see tesseract-ocr/tesseract#76).